.github/workflows/main.yml

```diff
@@ -0,0 +1,77 @@
+name: Test Square SVGs
```

Comment:
Change to "Test SVGs" instead
this Action does a lot more than checking for squares :frog: